sdk/communication/communication-phone-numbers/samples-dev/siprouting/listSipRoutingConfiguration.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { SipRoutingClient } from "@azure/communication-phone-numbers";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const buildClient = () => {
+  const connectionString =
```

Comment:
I would recommend inlining the client creation in the main sample and having connection string as a module level variable.

```ts
const connectionString = process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING ||
    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";


export async function main() {
  console.log("\n== SIP Routing Client Example ==\n");
  
  const client = new SipRoutingClient(connectionString);

//...
}


```

Reply:
Refactored.